sdcm/nemesis/monkey/sla.py

```
@@ -0,0 +1,229 @@
+    def format_error_for_sla_test_and_raise(error_events):
+        if any(error_events):
+            raise NemesisSubTestFailure(
+                "\n".join(f"Step: {error.step}. Error:\n - {error}" for error in error_events if error)
+            )
+
+
+class RemoveServiceLevelMonkey(SlaMonkeyBase):
+    """Remove service level while load is running, then re-create it."""
+
+    disruptive = True
+    additional_configs = []
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)